vendor.mod

```diff
@@ -32,6 +32,7 @@ require (
 	github.com/docker/go-units v0.4.0
 	github.com/docker/libkv v0.2.2-0.20211217103745-e480589147e3
 	github.com/docker/libtrust v0.0.0-20150526203908-9cbd2a1374f4
+	github.com/docker/swarmkit v1.12.0
```

Comment:
Can we avoid importing swarmkit v1 and v2 simultaneously?

Reply:
Resolved in 760200fdfd10007134a7fd36eaa3fdc77d2fd4fe.

Reply:
@AkihiroSuda, resolved in 760200fdfd10007134a7fd36eaa3fdc77d2fd4fe.